__tests__/components/DurationInput.spec.ts

```diff
@@ -100,9 +102,43 @@ describe('components/DurationInput', () => {
         const chosenValue = '20000';
         const wrapper = getDurationInputWrapper({ showRelativeTime: true, targetAsset: 'namespace', value: '10000' });
         const component = wrapper.vm as DurationInputTs;
+
+        // Act:
         component.chosenValue = chosenValue;
 
         // Assert:
         expect(wrapper.emitted('input')[0][0]).toBe(chosenValue);
     });
+
+    // TODO test for validationRule
```

Comment:
aren't new tests testing validationRule? is TODO still needed?

Reply:
yes, they are testing; removed the TODO line.